### DISCUSSION TITLE
Manual IP assignment support?

### DISCUSSION THREAD

**Discussion by parasiteoflife** [netbirdio/netbird#1415] [Q&A / Support]
Hey, I see we have a box for the IP address when we browse the users and click on their names but the IP box is locked, is manual assignment not supported? Are there plans to support it?

Thank you.

**Comment by mlsmaycon** [comment]
Hello @parasiteoflife, the box is just informational; the IP address is static and doesn't change.  You can potentially change it via updating the store. What would be your use case for manually configuring a peer IP?

**Reply by Darin755** [reply]
On my netbird account the IPs do change from time to time. I'm not sure why this is but I've had to use the DNS entry for my reverse proxies.

**Reply by mlsmaycon** [reply]
Hello @Darin755, the only scenario this should be possible is on peers activated by a setup-key with ephemeral enabled...these peers would disconnect from management for over 10 minutes, and then they would be recreated with the same name.

**Reply by mlsmaycon** [reply]
Can you confirm your case?

you should event logs in the activity tab as well

**Comment by Darin755** [comment]
> Hello @Darin755, the only scenario this should be possible is on peers activated by a setup-key with ephemeral enabled...these peers would disconnect from management for over 10 minutes, and then they would be recreated with the same name.

The Netbird software is installed natively and I don't think the peers are ephemeral. I haven't had the IP addresses change in a while now so it possible that this was a bug that was fixed. When it happened I assumed that the IPs were dynamic so I didn't think much of it. I must of just encountered a edge case (I will reply to this comment if it happens again)

Edit: I found the date when I had a issue but in the activity log is empty around that time. For reference, it was Oct 21
